frontend/plugins/accounting_ui/src/modules/adjustments/inventories/components/AdjustInventoryDetail.tsx

```
@@ -176,3 +177,71 @@ export const AdjustInventoryDetail = () => {
+          <DatePicker
+            value={adjustInventory?.beginDate}
+            onChange={() => null}
+            className="h-8 flex w-full"
+            disabled={true}
+          />
+        </span>
+        <span className="text-accent-foreground">{'->'}</span>
+      </div>
+      {days.map((day) => {
+        return (
+          <Tooltip delayDuration={0}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)